### PR TITLE
Update i18n

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Update i18n in extension point.

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: extension_point_creditcontrol_checkout/react/locales/en-US.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "verifyCheckingAccount_404": "No Checking Account found for your email.",
   "fetchAccountById_2003": "Account not found.",
   "fetchDeadlines_400": "Error fetching deadlines.",
+  "fetchDeadlines_404": "Deadlines not found",
```
Comment (Caio-Batista) [inline extension_point_creditcontrol_checkout/react/locales/en-US.json]: Falta o ponto final.

File: extension_point_creditcontrol_checkout/react/locales/es-AR.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   "verifyCheckingAccount_404": "No se encontró una Checking Account para su correo electrónico.",
   "fetchAccountById_2003": "Cuenta no encontrada.",
   "fetchDeadlines_400": "Error al buscar los plazos.",
+  "fetchDeadlines_404": "Parcelamientos no encontrados",
```
Comment (Caio-Batista) [inline extension_point_creditcontrol_checkout/react/locales/es-AR.json]: Ditto

File: extension_point_creditcontrol_checkout/react/locales/pt-BR.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
   "verifyCheckingAccount_404": "Nenhum Checking Account encontrado para seu e-mail.",
   "fetchAccountById_2003": "Nenhuma conta encontrada.",
   "fetchDeadlines_400": "Erro ao buscar os prazos.",
+  "fetchDeadlines_404": "Parcelamentos não encontrados",
```
Comment (Caio-Batista) [inline extension_point_creditcontrol_checkout/react/locales/pt-BR.json]: Ditto

File: extension_point_creditcontrol_checkout/react/locales/en-US.json (hunk 1 above)
Comment (kevinch) [reply extension_point_creditcontrol_checkout/react/locales/en-US.json]: Coming in another pr.

File: extension_point_creditcontrol_checkout/react/locales/es-AR.json (hunk 2 above)
Comment (kevinch) [reply extension_point_creditcontrol_checkout/react/locales/es-AR.json]: Coming in another pr.

Comment (kevinch) [reply extension_point_creditcontrol_checkout/react/locales/es-AR.json]: Coming in another pr.

File: extension_point_creditcontrol_checkout/react/locales/pt-BR.json (hunk 3 above)
Comment (kevinch) [reply extension_point_creditcontrol_checkout/react/locales/pt-BR.json]: Coming in another pr.

**Pull request merged**